lab_3_generate_by_ngrams/main.py

```diff
@@ -129,6 +183,11 @@ def fill_from_ngrams(self, content: dict) -> None:
         Args:
             content (dict): ngrams from external JSON
         """
+        if isinstance(content, dict) and content:
```

Comment:
:O i in for in for in if.... Please, simplify

Reply:
Simplified